test/functional/http/post/0.tx.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+var node = require('../../../node');
+var shared = require('./shared');
```

Comment:
For this PR, I would leave as is, i.e. without. Then we should open separate PR to standardise across whole application.